motr/m0crate/crate_index.c

```diff
@@ -527,14 +654,14 @@ static struct m0_bufvec* idx_bufvec_alloc(int nr)
 static size_t cr_idx_w_get_value_size(struct cr_idx_w *w)
 {
 	int vlen;
-
-	if (w->wit->record_size < 0)
-		vlen = sizeof(struct m0_fid) +
-			cr_rand_pos_range(w->wit->max_record_size);
+	/** If value_size is random, generate it using rand() */
```

Comment:
```
/**
   *
   */
```
is only used as doxygen comments in data structure definition and function definition.

Here we just use /*  some comments. */

Reply:
Fixed.